.changeset/purple-clocks-impress.md

```diff
@@ -0,0 +1,17 @@
+---
+'braid-design-system': minor
+---
+
+---
+updated:
+  - Toggle
+---
+
+**Toggle:** Add `bleedY` prop
+
+Introduces the `bleedY` prop, allowing you to remove excess vertical space created by the toggle handle.
```

Comment:
Worth being consistent with the docs which mention a "toggle input", rather than a "toggle handle".

```suggestion
Introduces the `bleedY` prop, allowing you to remove excess vertical space created by the toggle input.
```